backend/app.js

```diff
@@ -1,21 +1,79 @@
-const express = require('express');
-const cors = require('cors');
+// Import required modules
+require("dotenv").config();
```

Comment:
use the `.env` in root dir

Reply:
i don't get

Reply:
you can add the path to the `config` function, ideally this path should point to the `.env` in the project's root directory.

```require('dotenv').config({ path: '/custom/path/to/.env' })```

Reply:
okay